ucsb-cs156-f24/proj-rec-f24-06
# CL: Create Page for RequestTypeCreate plus tests and stories

Storybook: https://6734fdad790fe98dc6bf77f1-xbzqlvttjv.chromatic.com/?path=/story/components-requesttype-requesttypeform--create

Dokku deployment: https://proj-rec.dokku-06.cs.ucsb.edu/

NOTE TO STAFF: APPROVE after PR #77 This branch was created off of cl-reqtypes-placeholders because the work there was needed to do this work. After that PR is approved, rebase to branch from main instead.

This PR finishes the create page for the RequestType component. It also adds the corresponding tests and story files. These tasks continue the process of adding RequestType to the frontend. Ultimately, this works towards having RequestType fully integrated into our application

To Test:
The page should look like this at first:
![Screenshot 2024-11-23 at 5 21 58 PM](https://github.com/user-attachments/assets/a81745d3-3d5b-4fdf-9692-5084d545749d)

Don't input anything, just press create, it should look like this:
![Screenshot 2024-11-23 at 5 22 03 PM](https://github.com/user-attachments/assets/5ea8f39d-f7f2-4a8f-bb1e-430b4e847c34)

After you fill out all fields and press create, there should be a pop up on the top right similar to the one below:
![Screenshot 2024-11-23 at 5 22 11 PM](https://github.com/user-attachments/assets/b9a5b3a4-1581-4a02-afec-bb09f3b9aed3)

Related to Issue #42 and Closes #78
